### ISSUE TITLE
Media Uploads API

### ISSUE THREAD

**Issue by cadecairos** [mozilla/api.webmaker.org#8]
Questions:
* Images only?
* Where will we store them?
* What format?
* size limits?
* CDN?

**Comment by xmatthewx** [comment]
v1 is images only 
 - Image - JPG, PNG, GIF, SVG should be sufficient. Animated GIFs will be inevitable. Limit size accordingly.
 - Audio - Unlikely before Q4. Could be very interesting to play with on our platform, adding loops to pages, effects to nav transitions.   
 - Video - Unlikely before Q4. I wonder if we'd play with embeds before introducing video support.

**Comment by xmatthewx** [comment]
Stating the obvious: We will want to add some metrics to track abuse.

**Comment by thisandagain** [comment]
This is top priority for the beta SOW and we need some sort of implementation ready for testing before the end of this sprint. @cadecairos how would you like us to kick this off? Happy to jump on a call any time this week.

**Comment by cadecairos** [comment]
@thisandagain I'd like to develop a strategy for this ASAP so I get get started on an implementation. Got time today?

**Comment by cadecairos** [comment]
This looks handy https://devcenter.heroku.com/articles/s3-upload-node

**Comment by jbuck** [comment]
Yeah, that looks better than the pile of node modules I concocted writing a PR for webmaker app. But if they're a helpful reference at all: https://github.com/jbuck/s3-post-policy

**Comment by thisandagain** [comment]
To get around this we have been storing images in the `element` table itself. It is working well enough for now, but I'd like to sync up again on this once login is landed.

**Comment by cadecairos** [comment]
With login being delayed until at least mid-next week, what are we doing here?

**Comment by thisandagain** [comment]
@cadecairos Good question. Let's talk about this tomorrow at stand-up.
